app/dojo.py

```diff
@@ -0,0 +1,298 @@
+
+from os.path import isfile, getsize
+from random import choice
+
+from termcolor import cprint
+
+from .room import LivingSpace, Office
+from .person import Fellow, Staff
+
```

Comment:
Consider imports in the format `app.dojo`, they are cleaner. Check this [pep8 imports](https://www.python.org/dev/peps/pep-0008/#imports)